### PR TITLE
update toast notification usage docs

### PR DESCRIPTION
**Pull request by kingtraceyj**
Updating toast notification usage docs to reflect changes for [v11 update ](https://github.com/carbon-design-system/carbon/issues/8703)

### Added
- Types of toasts

### Updated toast notification section
- Actions
- Dismissal

### CODE REVIEW

File: src/pages/components/notification/usage.mdx [hunk 1]
```diff
@@ -235,7 +235,19 @@ critical for a user to read or interact with the notification.
 
 ## Toast notifications
 
-### Formatting
+There are two types of toast notifications—polite and assertive. Polite toast
+notifications are used to display a message to the user that does not interrupt
+keyboard and screen reader users current task and then queues the notification
```
Comment (jillianhowarth) [inline src/pages/components/notification/usage.mdx:240]:
```suggestion
keyboard and screen reader users' current task and then queues the notification
```

File: src/pages/components/notification/usage.mdx [hunk 2]
```diff
@@ -235,7 +235,19 @@ critical for a user to read or interact with the notification.
...
+until whatever task the user is currently working on is complete. Assertive
+toast notifications interrupt a users task when triggered and are more intrusive
```
Comment (jillianhowarth) [inline src/pages/components/notification/usage.mdx:242]:
```suggestion
toast notifications interrupt a user's task when triggered and are more intrusive
```

File: src/pages/components/notification/usage.mdx [hunk 3]
```diff
@@ -235,7 +235,19 @@ critical for a user to read or interact with the notification.
...
+to keyboard and screen reader users.
+
+| Type      | Purpose                                                                                                                                                                                                                                |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Polite    | Polite toast notifications are used to display a message to the user that does not interrupt keyboard and screen reader users current task. These notifications contain action buttons and are dismissed by the user.                  |
```
Comment (jillianhowarth) [inline src/pages/components/notification/usage.mdx:247]:
```suggestion
| Polite    | Polite toast notifications are used to display a message to the user that does not interrupt keyboard and screen reader users' current task. These notifications contain action buttons and are dismissed by the user.                  |
```

File: src/pages/components/notification/usage.mdx [hunk 4]
```diff
@@ -245,13 +257,30 @@ critical for a user to read or interact with the notification.
 </Column>
 </Row>
 
-#### Action
+#### Action (polite only)
+
+Polite toast notifications allow for interactive elements. Only one action
+button is allowed within polite toast notifications.  Action buttons should
```
Comment (jillianhowarth) [inline src/pages/components/notification/usage.mdx:263]:
```suggestion
button is allowed within polite toast notifications. Action buttons should
```

**Review by jillianhowarth — APPROVED** [review]
Looks great! 🙂

**Review by tw15egan — APPROVED** [review]

**Review by emyarod — COMMENTED** [review]
looks good to me after content updates

**Review by johnbister — APPROVED** [review]

**Pull request merged**